### DISCUSSION TITLE
Photoprism android app connection (error)

### DISCUSSION THREAD

**Discussion by xseph12** [photoprism/photoprism#4016] [Q&A] [answered]
Hi 
I have a problem using photoprism android app.

Photoprism is currently install in my Asustor Nas and i can access the site using my computer browser. But when i try to setup the photoprism in my Android device i encountered this error:
"Library is not accessible:ssl peer unverified exception :..."

I used the address (https://nasip:32770) which i can open using the computer browser.

Anyone encountered same problem and solve it?

**Comment by graciousgrey** [comment, marked as answer]
Hey :wave: 

You can find the repo of the android app here: https://github.com/Radiokot/photoprism-android-client

@Radiokot might be able to help you with this :)

**Reply by Radiokot** [reply]
Thanks, Theresa, this is to be discussed further [in the gallery repository](https://github.com/Radiokot/photoprism-android-client/discussions/119).
